opennms-doc/guide-admin/src/asciidoc/text/operation/jmx-config-generator.adoc

```diff
@@ -1,59 +1,252 @@
+=== JMX Config Generator
 
-// Allow GitHub image rendering
-:imagesdir: ../../images
+The _JMX Config Generator_ is a revised and improvised version of the original _JmxConfigTool_.
+It provides a fast and easy way to generate a valid and functional `jmx-datacollection-config.xml` file from a running Java application.
+Based on this generated configuration file it can also generate a `snmp-graph.properties` file with matching graph definitions.
 
-=== JmxConfigTool
-The JmxConfigTool provides a fast and easy way to generate a proper _jmx-datacollection-config_ from a running Java application.
-Based on the generated configuration file it can also provide a set of _snmp-graph_ configurations.
+==== Installation
+From current _OpenNMS_ versions 1.12 and later the _JMX Config Generator_ is included as a separate installable package.
```

Comment:
Sine _OpenNMS_ version 1.12 the _JMX Config Generator_ is included as a separate installable package.